### DISCUSSION TITLE
How to use an object created in Python in an R script and vice versa?

### DISCUSSION THREAD

**Discussion by cordeiroitamar** [posit-dev/positron#5348] [Q&A]
Suppose a DataFrame created in R:
```
library(tidyverse)

df_r = tibble('name' = c('Charles','Andrew','Joachim'), 'age' = c(12,15,13))
```
Is it possible to call this object in a Python script to work with it?

And the opposite? If I create an object in Python, how can I call it in an R script?
```
import pandas as pd

df_py = pd.DataFrame({
    'city': ['Paris', 'Tokyo', 'New York'],
    'country': ['France', 'Japan', 'USA']
})
```

**Comment by singerep** [comment]
I'm not a Positron developer so take this with a grain of salt, but I am under the impression that Positron does not have any special functionality to do this. There are libraries that do something like this, though, the most popular (from what I understand) being the R package [reticulate](https://rstudio.github.io/reticulate/).

**Reply by juliasilge** [reply]
Positron does have some special (currently experimental) handling for reticulate Python sessions; check out https://github.com/posit-dev/positron/discussions/3920#discussioncomment-10808127 for how to turn this feature on and check out how it might work for you.
